src/app/components/shared/menu/website-status-indicator/website-status-indicator.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from "@angular/core";
+import { WebsiteStatusService } from "@baw-api/website-status/website-status.service";
+import { websiteStatusMenuItem } from "@components/website-status/website-status.menu";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { StrongRoute } from "@interfaces/strongRoute";
+import { WebsiteStatus } from "@models/WebsiteStatus";
+import { takeUntil } from "rxjs";
+
+@Component({
+  selector: "baw-website-status-indicator",
+  template: `
+    <a
+      *ngIf="!isWebsiteHealthy"
+      class="block nav-link text-danger text-nowrap"
+      [strongRoute]="websiteStatusRoute"
+      [ngbTooltip]="
+        'The website is currently experiencing a temporary loss of services. ' +
+        'Some functionality may be reduced.'
```

Comment:
```suggestion
        '{{ brandName }} is experiencing a temporary loss of services. ' +
        'Some parts of the website may not work.'
```

(not correct syntax)